config/homebin/vagrant_provision

```diff
@@ -29,11 +31,11 @@ else
 fi
 
 vvv_info " * Restarting MariaDB service"
-if sudo service mariadb status > /dev/null; then
-  sudo service mariadb restart
-else
-  sudo service mariadb start
-fi
+if sudo service "${mysql_service_name[@]}" status > /dev/null; then
+    sudo service "${mysql_service_name[@]}" restart
+  else
+    sudo service "${mysql_service_name[@]}" start
+  fi
```

Comment:
I'd like to wrap all this in a helper function as we have similar stuff in the vagrant up and provision hooks as well as lots of provisioners and it's super repetitive

Reply:
reverted this shenanigans as well https://github.com/Varying-Vagrant-Vagrants/VVV/pull/2739#discussion_r1821351163